src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,16 +1,36 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
+    elements = len(arrA) + len(arrB) 
+    merged_arr = [0] * elements # creates a list of zeros in the length of arrA + arrB
+    # Let's layout the base case
+    if arrA == []:
+      return arrB
+    if arrB == []:
+      return arrA  
+    if arrA[0] <= arrB[0]:
+        # here I'm directly overwriting the elements of merged_arr
+        merged_arr = [arrA[0]] + merge(arrA[1:], arrB) # the 1st element of + call the fn
```

Comment:
Wow I'm really impressed you used recursion here to merge! You really jumped into recursion. I like this line of code. The solution code used a for loop and a bunch of if statements. I'm not sure which is more efficient but I like the way you implemented this.

Reply:
I worked for hours trying to run a for loop. I couldn't quite get their because I didn't know how to take merged_arr into account. I thought the assignment was about using recursion method so tried it little differently. I kept breaking because I hadn't included a base case! But thanks..I'll try to do it using for loop (if I get time). :-)